Makefile

```diff
@@ -26,7 +26,13 @@ OS     ?= $(shell go env GOOS)
 
 HELM_VERSION ?= 3.12.3
 KUBEBUILDER_TOOLS_VERISON ?= 1.28.0
+KUBECTL_VERSION ?= 1.28.2
```

Comment:
note: The kubectl version was previously updated dynamically by fetching the latest version from a remote URL. We should fix all dependencies so that for a given commit it's clear what versions of dependencies are tested together.